sw/device/lib/dif/dif_sensor_ctrl.h

```diff
@@ -12,13 +12,129 @@
  */
 
 #include <stdint.h>
-
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_base.h"
 #include "sw/device/lib/dif/autogen/dif_sensor_ctrl_autogen.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif  // __cplusplus
 
+/**
+ * An event, identified by a numeric id.
+ *
+ */
+typedef uint32_t dif_sensor_ctrl_event_idx_t;
+
+/**
+ * A vector of AST events, where each bit represents one event.
+ *
+ */
+typedef uint32_t dif_sensor_ctrl_events_t;
+
+/**
+ * IO power status
```

Comment:
Something like... "where each bit represents whether a particular rail is up." ?

Reply:
added